### ISSUE TITLE
Accessibility: brand image is not labeled

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/global-nav#302]
The brand image brings up the accessibility error "Content with images must be labeled."  When there is a brand title, this labeling is redundant. The text leaf with the title is also redundant because the encompassing span has an aria-label.

A simplification would be to add the "aria-hidden" attribute to the three elements with classes "brand-image", "esri-header-account-image", and "esri-header-brand-text", respectively.
